lib/index.js

```diff
@@ -317,15 +333,13 @@ internals.getParamsData = function (param, name) {
             });
         }
 
-        ['includes', 'excludes', 'items'].forEach(function (rule) {
-
-            if (param[rule]) {
-                data.rules[internals.capitalize(rule)] = param[rule].map(function (ruleType) {
-
-                    return internals.getParamsData(ruleType);
-                });
-            }
-        });
+        // If we have only one specific rule then set that to our type for
```

Comment:
This is to try to simplify the display for cases like this `Joi.string.guid()` to just display guid rather than string with a drawer for guid.